exercises/diffie-hellman/Cargo-example.toml

```diff
@@ -5,3 +5,6 @@ version = "0.1.0"
 
 [dependencies]
 rand = "0.3.18"
+
+[features]
+big-primes = []
```

Comment:
Please ensure that all text files end with a newline. I see that the original `Cargo.toml` didn't, which was an error of review on our part.

Reply:
Also noticed that and decided to stick with the original